docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/Running-Fast-Mode-Simulations.rst

```diff
@@ -0,0 +1,137 @@
+.. _FIREAXE-FAST-MODE:
+
+Running Fast Mode Simulations
+===================================
+
+In this section, we provide a step-by-step tutorial on how to partition a
+*RocketTile* out from the *SoC* and run fast-mode simulations using 2 FPGAs on EC2 F1.
+This assumes that you have completed the :ref:`AWS F1 Getting Started Guide<AWS-F1-Getting-Started-Guide>` and that you are familiar with
+the FireSim workflow.
+
+1. Building Partitioned Sims : Setting up FireAxe Target configs
+----------------------------------------------------------------
+
+To build bitstreams for partitioned simulations, we need specify which modules we want to partition out. 
+We can do that in `sim/firesim-lib/src/main/scala/configs/FireAxeTargetConfigs.scala`.
+
+.. literalinclude:: ../../../sim/firesim-lib/src/main/scala/configs/FireAxeTargetConfigs.scala
+   :language: scala
+   :start-after: DOC include start: F1 Rocket Partition
+   :end-before: DOC include end: F1 Rocket Partition
+
+The `WithPartitionGlobalInfo` fragment takes in an argument of type `Seq[Seq[String]]`
```

Comment:
Code formatting requires "``" instead of "`" everywhere. This needs fixing everywhere.